.storybook/config.js

```diff
@@ -8,7 +8,16 @@ import '../plugins/vue-filters'
 import '../assets/scss/style.scss';
 
 Vue.use(BootstrapVue);
-
+Vue.prototype.$t = function () {}
+Vue.component('NuxtLink', {
+  props:   ['to'],
+  methods: {
```

Comment:
What is this method for?

Reply:
This is so Nuxt Links work within Storybook otherwise it breaks.

Reply:
apologies, miss read this comment. Method is so that it can work in conjunction with StoryBook actions. Some info about it here https://medium.com/js-dojo/a-guide-on-using-storybook-with-nuxt-js-1e0018ec51c9#3977

For some reason the action console isnt showing... I will investigate

Reply:
StoryBook actions now work so you can see the method at work now :)

Reply:
Understood.